### PR TITLE
Features/directives/checklist

### PR DESCRIPTION
**Pull request by skazi0**

### CODE REVIEW

**Comment by bisht-richa** [comment]
Looks good.

File: assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                check2: { status: true },
+                check3: { status: false }
+            };
+            scope.$digest();
+        });
+
+        it('should contain a list with one item for each check', function () {
+            var uls = directiveElement.find('ul');
+            expect(uls.length).toEqual(1);
+            expect(angular.element(uls[0]).children().length).toEqual(_.size(scope.testChecks));
```
Comment (santiph) [inline assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js:60]: Shouldn't `_.size(scope.testChecks)` be `_.size(scope.testChecks.length)` instead?

Comment (skazi0) [reply assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js:60]: No. `scope.testChecks` is an object. If it was an array we would use `scope.testChecks.length` directly. To count properties of an object I chose `_.size(object)`.

File: assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.jade [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ul.check-list
     li(ng-repeat="(checkKey, check) in checklist",
-        translate='',
+        check-key='{{checkKey}}',
```
Comment (santiph) [inline assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.jade]: checkKey won't change here, please avoid two way data binding

File: assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        });
+
+        it('should have all items with labels maching the model', function () {
+            var lis = directiveElement.find('li');
+            _.each(lis, function (li) {
+                var itemElement = angular.element(li)
```
Comment (santiph) [inline assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js]: Missing semicolon.
Also, please use one `var` statement to declare multiple variables

**Review by santiph — CHANGES_REQUESTED** [review]
Please fix the var statement and consider this PR approved. Good job! 👍

**Comment by santiph** [comment]
Approved

**Review by santiph — APPROVED** [review]

**Pull request merged**